<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComputerDialog</class>
 <widget class="QDialog" name="addComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="computer_label_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="computer_label_yearbuilt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Built</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_computer_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="add_new_computer_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#550000;&quot;&gt;Add a new computer&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_yearbuilt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_add_newComputer_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error_computer_yearbuilt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_newComputer_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="computer_labe_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_electronic">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Electronic</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_mechatron">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechatron</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_transistor">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Transistor</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_other">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_wasBuilt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Was Built</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_computer_name</tabstop>
  <tabstop>input_computer_yearbuilt</tabstop>
  <tabstop>add_newComputer_button</tabstop>
  <tabstop>cancel_add_newComputer_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
